documentation/src/main/asciidoc/ch12.asciidoc

```diff
@@ -677,35 +677,41 @@ validator factory still is referenced by application code.
 ====
 
 [[section-getter-property-selection-strategy]]
-=== Getter property selection strategy configuration
+=== Customizing the getter property selection strategy
 
-When a regular validation operation is performed on an object it's properties get validated. Per
-JavaBean specification, a property can either be a field or a getter. Hibernate Validator uses
-next set of rules to determine if a method is a getter or not:
+When a bean is validated by Hibernate Validator, its properties get validated. A property can either
+be a field or a getter.
+By default, Hibernate Validator respects the JavaBeans specification and considers a method as a getter as soon
+as one of the condition below is true:
 
-- a method name starts with `get`, it has a nonvoid return type and has no parameters
-- a method name starts with `is`, has a return type of `boolean` and has no parameters
-- a method name starts with `has`, has a return type of `boolean` and has no parameters (this
+- the method name starts with `get`, it has a non-void return type and has no parameters;
+- the method name starts with `is`, has a return type of `boolean` and has no parameters;
+- the method name starts with `has`, has a return type of `boolean` and has no parameters (this
 is a Hibernate Validator specific rule and is not mandated by the JavaBeans specification)
 
-If any of the statements above are true - method is considered a getter.
+While there rules are usually appropriate when following the classic JavaBeans convention, it might happen,
+especially with code generators, that the JavaBeans naming convention is not followed and that the getters'
+name are following a different convention.
 
-When working with objects that do not follow JavaBean specification rules on getters. These rules
-should either be redefined or extended, in order to be able to fully validate the object. An example
-of such objects can be the ones constructed from classes that follow fluent design pattern (i.e. <<example-using-fluent-api-pattern>> ).
+In this case, the strategy for detecting getters should be either extended or redefined
+in order to fully validate the object.
+
+A classic example of this requirement is when the classes follows a fluent naming convention.
```

Comment:
Maybe comma at the end or capitalize the first letter ('As')

Reply:
Good catch, it was supposed to be a comma!